synapse/logging/tracing.py

```diff
@@ -202,86 +204,35 @@ def set_fates(clotho, lachesis, atropos, father="Zues", mother="Themis"):
 
 # Helper class
 
+# Always returns the value given for any accessed property
+class _DummyLookup(object):
+    def __init__(self, value):
+        self.value = value
 
-class _DummyTagNames:
-    """wrapper of opentracings tags. We need to have them if we
-    want to reference them without opentracing around. Clearly they
-    should never actually show up in a trace. `set_tags` overwrites
-    these with the correct ones."""
-
-    INVALID_TAG = "invalid-tag"
-    COMPONENT = INVALID_TAG
-    DATABASE_INSTANCE = INVALID_TAG
-    DATABASE_STATEMENT = INVALID_TAG
-    DATABASE_TYPE = INVALID_TAG
-    DATABASE_USER = INVALID_TAG
-    ERROR = INVALID_TAG
-    HTTP_METHOD = INVALID_TAG
-    HTTP_STATUS_CODE = INVALID_TAG
-    HTTP_URL = INVALID_TAG
-    MESSAGE_BUS_DESTINATION = INVALID_TAG
-    PEER_ADDRESS = INVALID_TAG
-    PEER_HOSTNAME = INVALID_TAG
-    PEER_HOST_IPV4 = INVALID_TAG
-    PEER_HOST_IPV6 = INVALID_TAG
-    PEER_PORT = INVALID_TAG
-    PEER_SERVICE = INVALID_TAG
-    SAMPLING_PRIORITY = INVALID_TAG
-    SERVICE = INVALID_TAG
-    SPAN_KIND = INVALID_TAG
-    SPAN_KIND_CONSUMER = INVALID_TAG
-    SPAN_KIND_PRODUCER = INVALID_TAG
-    SPAN_KIND_RPC_CLIENT = INVALID_TAG
-    SPAN_KIND_RPC_SERVER = INVALID_TAG
+    def __getattribute__(self, name):
+        return object.__getattribute__(self, "value")
 
 
+# These dependencies are optional so they can fail to import
+# and we
 try:
-    import opentracing
-    import opentracing.tags
+    import opentelemetry
+    import opentelemetry.trace
+    import opentelemetry.sdk.trace
+    import opentelemetry.sdk.trace.export
+    import opentelemetry.sdk.resources
+    import opentelemetry.semconv.trace
+    import opentelemetry.exporter.jaeger.thrift
```

Comment:
Do we want to use `thrift` or `protobuf`? The docs are [finally](https://github.com/open-telemetry/opentelemetry.io/pull/1563/files) [corrected](https://github.com/open-telemetry/opentelemetry.io/pull/1566/files) on the proper imports to use.

Reply:
Checking in our existing Jaeger, we use `internal.span.format | "proto"` currently

Reply:
Seems like thrift over UDP is the thing to use so I'll leave it, https://www.jaegertracing.io/docs/next-release/apis/